concordium-consensus/tests/consensus/ConcordiumTests/KonsensusV1/Types.hs

```diff
@@ -34,6 +34,10 @@ genBlsSignature = flip Bls.sign someBlsSecretKey . BS.pack <$> vector 10
 genQuorumSignature :: Gen QuorumSignature
 genQuorumSignature = QuorumSignature <$> genBlsSignature
 
+-- |Generate a block hash for a genesis
+genGenesisBlockHash :: Gen BlockHash
+genGenesisBlockHash = BlockHash . Hash.Hash . FBS.pack <$> vector 32
```

Comment:
Could rename to `genBlockHash`. Documentation should note this is an arbitrary hash value from the range of hashes.